onto_spread_ed/templates/base.html

```diff
@@ -55,7 +55,7 @@
         <ul class="navbar-nav me-auto mb-2 mb-lg-0">
         <li class="nav-item"><a class="nav-link active" aria-current="page" href="/">Home</a></li>
```

Comment:
This should use the prefix as well.

Reply:
Changed to this : href="{{url_for('main.home')}}"
because of this code in main:
bp = Blueprint("main", __name__, template_folder="../templates")


@bp.route('/')
@bp.route('/home')
@verify_logged_in
def home():